tests/integration/components/gleam-logo-test.ts

```
@@ -0,0 +1,224 @@
+declare global {
+  interface Window {
+    Rive: typeof MockRive;
+  }
+}
+
+// Mock Rive class for testing
+class MockRive {
+  @tracked canvas: HTMLCanvasElement;
+  @tracked stateMachineNames: string[] = ['State Machine 1', 'State Machine 2'];
+  @tracked isLoaded = false;
+  private _listeners: Map<string, Function[]> = new Map();
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)